Re-enable dpres_plevel testing

khallock opened the pull request:
The "unstable and untested" warning still exists in the Cython code for `dpres_plevel`. The warning should be removed once the problem with `dpres_plevel` is identified and fixed.

erogluorhan commented:
I might have missed your intention here: Did you meant to get this PR approved and merged immediately? My understanding is that this PR should be a "draft" PR instead, and we should wait until the problem with `dpres_plevel` is solved. In other words, we should not merge `test_dpres_plevel.py` immediately. Am I wrong at some point?

khallock commented:
@erogluorhan you are correct, sorry for the confusion. I intended for this to be a draft PR, but it's not possible to change that once the PR is created; I'll put a "work-in-progress" label on it now instead.

I added a new `NcompWarning` in the Cython code for `dpres_plevel` and removed the `test_dpres_plevel` tests from the develop branch. This PR re-enables those tests, and should theoretically experience intermittent failures for those tests (although it hasn't yet) until we identify and fix the problem. I intend to use this PR for new commits related to fixing this bug. Once we fix the issue and eliminate the test failures, we can remove the warning from the Cython code and merge this PR.

erogluorhan commented:
@khallock Great, thanks for the clarification! All of my confusion is addressed now and "work-in-progress" label works well.

khallock commented:
Migrated to NCAR/geocat-ncomp#5
